TestFramework/ExtensionMethods.cs

```diff
@@ -433,5 +433,36 @@ public static void WaitForElementVisible(IWebDriver driver, By by, int? timeInSe
                 count++;
             }
         }
+
+        public static void ClearText(this IWebElement element, int? timeInSec = null)
```

Comment:
I'm curious, where are you using this?  It might be good to have it where you enter a phone number in bookings as it's appending a new value there.

Reply:
Similar to phone number, it's appending to Organisation value also. Sometimes simply calling element.Clear() is not clearing the input filed. That's why I have created a new extension method. I will use for Phone number too

Reply:
Thanks